zksync_sdk/serializers.py

```diff
@@ -71,6 +75,33 @@ def integer_to_float(integer: int, exp_bits: int, mantissa_bits: int, exp_base:
     return result
 
 
+def integer_to_float_up(integer: int, exp_bits: int, mantissa_bits: int, exp_base) -> List[int]:
+    max_exponent_power = pow(2, exp_bits) - 1
+    max_exponent = pow(exp_base, max_exponent_power)
+    max_mantissa = pow(2, mantissa_bits) - 1
+
+    if integer > max_mantissa * max_exponent:
+        raise WrongIntegerError("Integer is too big")
+
+    exponent = 0
+    exponent_temp = 1
+    while integer > max_mantissa * exponent_temp:
+        exponent_temp = exponent_temp * exp_base
+        exponent += 1
+
+    # mantissa = integer // exponent_temp
+    mantissa = int(ceil(integer / exponent_temp))
+    if not divmod(integer, exponent_temp):
+        mantissa += 1
+
+    encoding = num_to_bits(exponent, exp_bits) + num_to_bits(mantissa, mantissa_bits)
+    encoding.reverse()
+    result = bits_into_bytes_in_be_order(encoding)
+    result.reverse()
```

Comment:
The same here, please, make it consistent with `integer_to_float`

Reply:
but what about that the construction is better than modification. I mean to say:
```encoding = num_to_bits(exponent, exp_bits) + num_to_bits(mantissa, mantissa_bits)``` 
is better than:
```
    data = []
    data.extend(num_to_bits(exponent, exp_bits))
    data.extend(num_to_bits(mantissa, mantissa_bits))
```

Reply:
Ok, then do `encoding = num_to_bits(exponent, exp_bits) + num_to_bits(mantissa, mantissa_bits)` in both methods

Reply:
Ok